config/v1/types_node.go

```diff
@@ -42,6 +42,14 @@ type NodeSpec struct {
 	// the status and corresponding reaction of the cluster
 	// +optional
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
+
+	// eventedPleg enables the event based PLEG between the kubelet and CRI-O
+	// Reference: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3386-kubelet-evented-pleg/README.md
```

Comment:
Is there a user friendly explanation we can include? A link to a KEP is quite a lot of context.

Is the intention when a user enables this feature that CRIO and Kubelet are both configured? Is there explicit configuration required for both? 

EventedPLEG is an upstream feature gate, what happens when that is enabled by default?